lib/rubocop/cop/rspec/rails/minitest_assertions.rb

```diff
@@ -60,7 +66,8 @@ class EqualAssertion < BasicAssertion
               refute_equal
             ].freeze
 
-            NODE_MATCHER_PATTERN = <<~PATTERN
+            # @!method self.minitest_assertion(node)
```

Comment:
I’m actually not sure if the correct YARD is your suggestion or
```suggestion
            # @!method minitest_assertion(node)
            # @!scope class
```

Reply:
I think you're right, but it still annoys `InternalAffairs/NodeMatcherDirective` 😞

Reply:
I opened an issue on RuboCop: https://github.com/rubocop/rubocop/issues/12665

For now, should we consider your PR ready to merge?

Reply:
yup works for me